packages/e2e-test-utils/src/open-publish-panel.js

```diff
@@ -2,9 +2,20 @@
  * Opens the publish panel.
  */
 export async function openPublishPanel() {
```

Comment:
`openPublishPanel` and `publishPost` were never updated to handle the multi-entity saving flow, so I had to do that.